net-fs/eos-client/eos-client-5.1.27.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="EOS Storage"
+HOMEPAGE="https://eos-web.web.cern.ch/eos-web/"
+SRC_URI="https://github.com/cern-eos/eos/archive/refs/tags/${PV}.tar.gz"
+LICENSE="GPL-3"
+
+SLOT="0"
+KEYWORDS="~amd64"
```

Comment:
Try to keep the layout as close to the ebuild skeleton as possible. (i.e. move `LICENSE` one line down)